ReactAndroid/src/main/java/com/facebook/react/views/image/ReactImageView.java

```diff
@@ -240,6 +287,22 @@ public void setFadeDuration(int durationMs) {
     // no worth marking as dirty if it already rendered..
   }
 
+  private float[] cornerRadii() {
+    float defaultBorderRadius = !CSSConstants.isUndefined(mBorderRadius) ? mBorderRadius : 0;
+
+    float topLeftRadius = mBorderCornerRadii != null && !CSSConstants.isUndefined(mBorderCornerRadii[0]) ? mBorderCornerRadii[0] : defaultBorderRadius;
+    float topRightRadius = mBorderCornerRadii != null && !CSSConstants.isUndefined(mBorderCornerRadii[1]) ? mBorderCornerRadii[1] : defaultBorderRadius;
+    float bottomRightRadius = mBorderCornerRadii != null && !CSSConstants.isUndefined(mBorderCornerRadii[2]) ? mBorderCornerRadii[2] : defaultBorderRadius;
+    float bottomLeftRadius = mBorderCornerRadii != null && !CSSConstants.isUndefined(mBorderCornerRadii[3]) ? mBorderCornerRadii[3] : defaultBorderRadius;
+
+    return new float[] {
```

Comment:
Allocations on android are pretty expensive (mainly because of the fact they need to be garbage collected, and still lots of android devices doesn't have a decent implementation of GC).

Looks like the pattern in which you use the value returned from here is that you call `cornderRadii`, then almost immediately you use the returned values to pass them to some other method (the important bit is that you use values from the array as opposed to passing the reference to that array). Also note that you only call this method from the UIThread. In this case you can easily avoid allocating this array and use a static array of size 4. The preferred way to do that would be to define the array at the top of the class, then change the sigatore of `cornerRadii` method to take output array as an argument. In places where you call the method, you can pass that static array, then retrieve values from it.